spec/features/preassembly_gis_raster_accessioning_spec.rb

```diff
@@ -85,8 +85,8 @@
     # sleep 1 # if you notice the project name not filling in completely, try this to
     #           give the page a moment to load so we fill in the full text field
     fill_in 'Project name', with: preassembly_project_name
-    select 'Pre Assembly Run', from: 'Job type'
-    select 'Geo', from: 'Content structure'
+    select 'preassembly', from: 'Job type'
+    select 'geo', from: 'batch_context_structure'
     fill_in 'Staging location', with: preassembly_bundle_dir
```

Comment:
note by by selecting form values instead of display text and form IDs instead of label, we can be a more resilient to changes in what we show the user